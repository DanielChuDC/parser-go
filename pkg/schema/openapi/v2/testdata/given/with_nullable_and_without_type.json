{
  "payload": {
    "nullable": true,
    "example": {
      "test": "example1"
    },
    "examples": [
      {
        "test": "example0"
      }
    ],
    "properties": {
      "test": {
        "nullable": true
      }
    },
    "additionalProperties": {
      "nullable": true
    }
  }
}