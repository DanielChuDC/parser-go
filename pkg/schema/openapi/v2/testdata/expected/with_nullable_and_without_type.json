{
  "payload": {
    "additionalProperties": {
      "type": [
        "null"
      ]
    },
    "type": [
      "null"
    ],
    "properties": {
      "test": {
        "type": [
          "null"
        ]
      }
    },
    "examples": [
      {
        "test": "example0"
      },
      {
        "test": "example1"
      }
    ]
  }
}